aquavision

```diff
@@ -1 +1 @@
-Hi Barbie
```

Comment:
🍆🍆🍆 But now I feel like a receptacle -- an empty vessel to be filled at his whim.